src/amo/components/Search/index.js

```diff
@@ -254,7 +251,7 @@ export function mapStateToProps(state) {
 // This ID does not need to differentiate between component instances because
 // the error handler gets cleared every time the search filters change.
 export const extractId = (ownProps) => {
-  return parsePage(ownProps.filters.page);
+  return ownProps.filters.page;
```

Comment:
Should this maybe return a more useful default than `undefined`?

Reply:
I discussed this issue with @kumar303 and I believe we decided that just returning undefined in all cases makes sense, because that's what's going to be passed to the API with this change.